wordclock/Makefile

```diff
@@ -27,12 +27,13 @@ EXTRA_COMPONENTS = extras/i2s_dma extras/ws2812_i2s extras/sntp extras/dhcpserve
 LIBS += m
 include ../esp-open-rtos/common.mk
 
-all: #$(FW_FILE)
-    # strip CRC from the end of the file. The Wordclocks software rejects the file size.
-	$(shell truncate -s -4 $(FW_FILE))
-	echo stripped CRC from $(FW_FILE) 
+strip_crc:
```

Comment:
Laat het strippen van de CRC er a.u.b. in zitten zoals het was. Voor do_flash maakt het niet uit of de CRC erachter zit of niet. Voor de OTA mag hij er niet achter. We krijgen lastige compattibiliteitsissues als we de esp-open-rtos aanpassen zodat hij de CRC accepteert. Volgens mij is de CRC  dus alleen nodig voor de windows tool. Maak daarvoor dan een kopie met _crc in de naam.

Reply:
Dat vind ik een prima oplossing!